content/pages/pension/application/527EZ.md

```diff
@@ -4,11 +4,12 @@ entryname: pensions
 layout: page-react.html
 ---
 <div id="main">
-  <nav class="va-nav-breadcrumbs">
-    <ul class="row va-nav-breadcrumbs-list columns" role="menubar" aria-label="Primary">
+  <nav aria-label="Breadcrumb" aria-live="polite" aria-relevant="additions text" class="va-nav-breadcrumbs js-visual"
+  id="va-breadcrumbs">
+    <ul class="row va-nav-breadcrumbs-list columns" id="va-breadcrumbs-list">
       <li><a href="/">Home</a></li>
       <li><a href="/pension/">Pension Benefits</a></li>
-      <li><a href="/pension/apply/">Application Process</a></li>
+      <li><a aria-current="page" href="/pension/apply/">Apply for Pension Benefits</a></li>
```

Comment:
@mnorthuis @bethpotts Would either of you mind taking a look at the text on this view? I changed it from "Application Process" to "Apply for Pension Benefits" to be more consistent with how we handle the text on /health-care/apply. Screenshots attached below, and will change back if this is the wrong text. Thank you!

---

![screen shot 2018-06-19 at 10 04 28 am](https://user-images.githubusercontent.com/934879/41607576-f2f79f76-73ab-11e8-92e2-32c136c66458.png)

---

![screen shot 2018-06-19 at 10 04 54 am](https://user-images.githubusercontent.com/934879/41607595-fd761d4c-73ab-11e8-98bd-48fd1fb0e9a5.png)

Reply:
@1Copenut this is an accurate change, thank you for checking!

Reply:
I think we should say, "Apply for Veterans Pension" since there's also an application for survivors pension. It's not an online form yet though.

Reply:
I agree with @bethpotts, good catch Beth!

Reply:
Excellent. This screen shot is from my local instance, will be pushed to the branch momentarily.

---

![screen shot 2018-06-19 at 11 01 02 am](https://user-images.githubusercontent.com/934879/41609450-649db9cc-73b0-11e8-86fb-ff06d59b0352.png)